transaction/core/Cargo.toml

```diff
@@ -10,13 +10,13 @@ aes = { version = "0.7.5", default-features = false, features = ["ctr"] }
 displaydoc = { version = "0.2", default-features = false }
 generic-array = { version = "0.14", features = ["serde", "more_lengths"] }
 hex_fmt = "0.3"
-hkdf = { version = "0.9", default-features = false }
+hkdf = "0.12"
```

Comment:
This is fine to stay as `0.12`, because transactions are (currently) using `Hkdf<Sha512>`.